platform/security/platform-security-core-api/src/main/java/ddf/security/principal/AnonymousPrincipal.java

```diff
@@ -18,11 +18,11 @@
 import org.apache.commons.lang.StringUtils;
 
 /**
- * Principal that designates a {@link ddf.security.Subject} as anonymous
+ * Principal that designates a {@link ddf.security.Subject} as guest
  */
 public class AnonymousPrincipal implements Principal {
```

Comment:
No reasons that I can think of. I believe we weren't renaming classes simply because we know that anonymous = guest, but we certainly could rename them ¯\_(ツ)_/¯